python/shotgun_desktop/upgrade_startup.py

```diff
@@ -106,9 +84,20 @@ def upgrade_startup(splash, sgtk, app_bootstrap):
     out_of_date = (latest_descriptor.get_version() != current_desc.get_version())
 
     if not out_of_date:
-        logger.debug("Desktop startup does not need upgrading. Currenty running version %s" % current_desc.get_version())
+        logger.debug(
+            "Desktop startup does not need upgrading. Currenty running version %s" % current_desc.get_version()
```

Comment:
i suggest changing it into a positive. it's easier to process cognitively. e.g.

"Desktop startup is up to date. Currently running version 1.2.3"